pkg/controller/llmisvc/DEV.md

```diff
@@ -418,16 +423,130 @@ spec:
+oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/tmp/final-pull-secret.json  
+
+# Create MirrorSet to pull prebuilt images 
+cat <<EOF| oc create -f -
+apiVersion: config.openshift.io/v1
+kind: ImageTagMirrorSet
+metadata:
+    name: stage-registry
+spec:
+    imageTagMirrors:
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh
+          source: registry.redhat.io/openshift-service-mesh
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh-tech-preview
+          source: registry.redhat.io/openshift-service-mesh-tech-preview
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh-dev-preview-beta
+          source: registry.redhat.io/openshift-service-mesh-dev-preview-beta
+---
+apiVersion: config.openshift.io/v1
+kind: ImageDigestMirrorSet
+metadata:
+    name: stage-registry
+spec:
+    imageDigestMirrors:
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh
+          source: registry.redhat.io/openshift-service-mesh
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh-tech-preview
+          source: registry.redhat.io/openshift-service-mesh-tech-preview
+        - mirrors:
+            - registry.stage.redhat.io/openshift-service-mesh-dev-preview-beta
+          source: registry.redhat.io/openshift-service-mesh-dev-preview-beta
+EOF
+
+# Deploy OSSM
+cat<<EOF |oc create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: istio-catalog
+  namespace: openshift-marketplace
+spec:
+  displayName: istio-catalog
+  image: brew.registry.redhat.io/rh-osbs/iib:1015285
+  publisher: grpc
+  sourceType: grpc
+EOF
+
+cat<<EOF|oc create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: servicemeshoperator3
+  namespace: openshift-operators
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: servicemeshoperator3
+  source: istio-catalog
+  sourceNamespace: openshift-marketplace
+  startingCSV: servicemeshoperator3.v3.1.0
+EOF
+
+oc wait --for=condition=ready pod -l control-plane=servicemesh-operator3 -n openshift-operators --timeout=300s
 
-**Install upstream ISTIO**
+
+oc create ns istio-cni  
+cat<<EOF|oc create -f -
+kind: IstioCNI
+apiVersion: sailoperator.io/v1
+metadata:
+  name: default
+spec:
+  namespace: istio-cni
+  version: v1.26.2
+EOF
+
+oc create ns istio-system
+cat<<EOF|oc create -f -
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  namespace: istio-system
+  updateStrategy:
+    type: InPlace
+    inactiveRevisionDeletionGracePeriodSeconds: 30
+  version: v1.26.2
+  values:
+    pilot:
+      env:
+        SUPPORT_GATEWAY_API_INFERENCE_EXTENSION: "true"
+        ENABLE_GATEWAY_API_INFERENCE_EXTENSION: "true"
```

Comment:
I think `ENABLE_GATEWAY_API_INFERENCE_EXTENSION` should be soon the only one available, lets put TODO to remove the `SUPPORT` - see https://github.com/istio/istio/pull/57099

Reply:
added comment